### PR TITLE
feat(lamp): add rhythms cicle

### PR DESCRIPTION
**Pull request by sunny-ali**
## Owner

- [x] Nickname(Tuya): Sophia
- [x] Intra-group review（组内审阅人员）：若叶，萧瑟

## Description

add rhythms circle

## Package

- [x] tuya-panel-lamp-sdk

## Type of change

Please select the relevant option.

- [x] New feature (non-breaking change which adds functionality)

## Self CheckList

Please check all options.

- [x] Intra-group review
- [x] ESLint lint
- [x] Example Run
- [x] Jest Run
- [x] No Grammer Error
- [x] No Spell Error
- [x] No Merge Conflict

### CODE REVIEW

File: example/src/scenes/Lamp/RhythmsCircle/index.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  },
+];
+const RhythmsCircleScene: React.FC = () => {
+  const [data, setData] = useState(defaultData);
+  const [data1, setData1] = useState(defaultData);
+
+  return (
+    <View style={styles.flex1}>
+      <>
+        <Text style={styles.text}>No more than</Text>
```
Comment (XiaooSe) [inline example/src/scenes/Lamp/RhythmsCircle/index.tsx:55]: 统一用TYText吧

Comment (XiaooSe) [reply example/src/scenes/Lamp/RhythmsCircle/index.tsx:55]: demo 多语言

File: example/src/i18n/strings.js [hunk 2]
```diff
@@ -68,5 +70,7 @@ module.exports = {
     TYLamp_animatedModal_darkHeaderTitle: 'dark header',
     TYLamp_animatedModal_cancelText: 'cancel',
     TYLamp_animatedModal_confirmText: 'confirm',
+    TYLamp_rhythms_tip: 'Node no more than',
```
Comment (paruki-gin) [inline example/src/i18n/strings.js:73]: code需要语义化

**Comment by XiaooSe** [comment]
LGTM

**Pull request merged**